### ISSUE TITLE
In-depth capture: Capture all links in tab mode too?

### ISSUE THREAD

**Issue by niilaT** [danny0838/webscrapbook#236]
Hi, cool extension!
I needed to do an in-depth capture of a website, so that all pages would be captured after being opened in browser tabs, because in the source mode some parts of the pages wouldn't initialize correctly.
I set the configuration to capture an open tab (or url); "tab" mode, depth value and address regex.

However, it seemed to me it downloaded just the specified first page in "tab" mode and then the other pages linked by it in "source" mode.

Does it have the feature to do an in-depth capture _opening also the linked pages in new tabs?_ If not, would be nice if that was an option.
Or perhaps I just didn't know how to use it correctly? :'D

(I did manage to make it 'kind of work' for my use case by swapping the captureDocumentOrFile call in capturer.captureUrl to captureRemote (mode set to "tab"). With the hack it successfully saved the same site for two(+ ?) times though (one would have been enough :P), and was quite slow since it recursively opened all already-visited links in new tabs, too.)

**Comment by danny0838** [comment]
@abreinig Yes, anything outside an opened tab is captured using source mode by default. You can change the behavior in JSON mode, however.

**Comment by abreinig** [comment]
@danny0838 That is exactly what I need. However, it doesn't work. Am I doing something wrong, I get no wait at all it just immediately opens the tab and captures it.

```
{
 "tasks": [
  {
   "url": "https://www.website.com/5A2",
   "title": "Website Title",
   "capture.remoteTabDelay": 10000,
   "mode": "tab"
  }
 ]
}
```

**Comment by abreinig** [comment]
@danny0838 Thank you, the prefill of Capture As cleared it up. This is amazing and does exactly what I wanted, thank you so much!
